php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -11,6 +11,8 @@
  * @link     https://www.github.com/aces/Loris-Trunk/
  */
 
+include_once 'BVL_Feedback_Panel.class.inc';
```

Comment:
The vendor/autoload file should automatically include the file if it's in php/libraries instead of htdocs